src/store/accounts/mutations.js

```diff
@@ -0,0 +1,59 @@
+import { Wallet, Address } from '@/class';
+import {
+  ADD_ADDRESS,
+  SET_ADDRESS,
+  ADD_WALLET,
+  SET_WALLET,
+  SET_HD_KEY,
+  SET_BALANCE,
+} from './mutations-types';
+
+const setAddress = (state, newAddress) => {
+  let address = newAddress;
+
+  if (!(address instanceof Address)) {
+    address = new Address(address);
+  }
+
+  state.address = address;
+};
+
+const setWallet = (state, wallet) => {
+  state.wallet = wallet;
+};
+
+const addWallet = (state, walletV3) => {
+  const wallet = new Wallet(walletV3);
+  const { address } = walletV3;
+  state.wallets = {
+    ...state.wallets,
+    [address]: wallet,
+  };
+};
+
+// Adds an empty wallet in order to view a public key
+const addAddress = (state, address) => {
+  state.wallets = {
```

Comment:
[The same story.](https://github.com/endpass/endpass-wallet/pull/222/files#diff-8b2f8c9b98a21a0dc73e55f3bf3ba057R29)